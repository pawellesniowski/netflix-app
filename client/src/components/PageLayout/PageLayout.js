import React from 'react';
import './PageLayout.scss';

const PageLayout = (props) => (
  <div className="page-layout">{props.children}</div>
);

export default PageLayout;
