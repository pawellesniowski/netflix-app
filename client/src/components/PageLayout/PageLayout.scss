@import "../../common/styles/colors";

* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

html {
  font-size: 10px;
}

.page-layout {
  min-height: 100vh;
  position: relative;
  padding-bottom: 5rem;
  background-color: $color-grey-mercury;
}


