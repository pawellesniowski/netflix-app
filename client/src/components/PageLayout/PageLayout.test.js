import React from 'react';
import {shallow} from "enzyme/build";
import PageLayout from "./PageLayout";

it("Renders PageLayout Component", () => {
  const component = shallow(<PageLayout/>);
  expect(component).toMatchSnapshot();
});
