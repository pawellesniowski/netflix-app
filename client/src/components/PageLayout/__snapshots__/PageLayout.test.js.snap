// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders PageLayout Component 1`] = `ShallowWrapper {}`;
